### DISCUSSION TITLE
Installed and running, where is sysop menu?

### DISCUSSION THREAD

**Discussion by dayzciv** [NuSkooler/enigma-bbs#344] [Q&A] [answered]
OK it is installed and running on a server..  I am logged in, applied, made first account, it shows me as sysop, user groups.  Perfect!  How do I access a sysop menu, however?

**Comment by NuSkooler** [comment, marked as answer]
There is not a "sysop" menu. What functionality are you looking for? Please see the docs regarding management/etc.

A Waiting For Caller (WFC) is in the works.

**Comment by IBBBS** [comment]
So I started a sysop menu but haven't quite figured it out yet. I have linux shell commands that I've created to upgrade, downgrade or delete users, and then I was going to use @execute in hjson under the menu area that I've built for Sysop menu. In the sysop menu, the sysop would enter user name ... and .... that's where I get stuck. I haven't figure out how to take that value and add it to the @execute <linux.sh> portion of the processing.  This is all being done in menu.hjson and may need to be moved to its own or menu-supporting module?

**Reply by NuSkooler** [reply]
The `@execute` syntax is only used in the Event Scheduler right now. If you want to run an arbitrary command you can treat it like a door, but `abracadabra.js` doesn't currently let you pass arbitrary arguments from another menu. Fairly easy to add, basically though.

But yes, you can also just make your own module and pass/do whatever you want from it.

**Reply by IBBBS** [reply]
So took two years before I replied to this - but blame it on me being old and senile. ^^^^ THIS (abracadabra) is the most awesome-ist. I was able to make a great sysop menu and abracadabra drives the engine behind it. PERFECT.  Thanks.

**Comment by IBBBS** [comment]
got it; thanks Bryan.
I crashed my BBS with the npm install LOL - bad sysop! bad!

On Thu, Feb 11, 2021 at 11:00 PM Bryan Ashby <notifications@github.com>
wrote:

> The @execute syntax is only used in the Event Scheduler right now. If you
> want to run an arbitrary command you can treat it like a door, but
> abracadabra.js doesn't currently let you pass arbitrary arguments from
> another menu. Fairly easy to add, basically though.
>
> But yes, you can also just make your own module and pass/do whatever you
> want from it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/NuSkooler/enigma-bbs/discussions/344#discussioncomment-361667>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASUPVM2CHT2MLJ3PRMYIKMTS6SRVRANCNFSM4V4UIWIQ>
> .
>


-- 
*// WESTWOOD BBS II // SYSOP // telnet WestwoodBBS.net 8888*

**Reply by NuSkooler** [reply]
If you need to be able to pass args from menu "A" to menu "B" which is a arbitrary door (e.g. a shell command or whatever) please open a ticket. I don't think it's too bad to implement.
